After wglc oct 25

talmi opened the pull request:
Updates following WGLC Comments
- Added clarifications about explicit Type I or Type II (Giuseppe).
- Explicit example of in-data-packet combined with active/passive, emphasizing that this is Hybrid Type I (Giuseppe/ Thomas).
- Various nits (Qin).

talmi committed:
Updates following Tim Chown review

Review comments:
https://mailarchive.ietf.org/arch/msg/opsawg/y0JPuKvChQMzRxv_las61a7I4Js/

talmi committed:
Updates based on review from Thomas Graf

Review from Thomas Graf:
https://mailarchive.ietf.org/arch/msg/opsawg/adB70UL415cYcmdnVbWLpVTe5iI/

talmi committed:
Updates based on review from  Chongfeng Xie

Changed the title of the section "Active, Passive, and Hybrid OAM".
Updated acknowledgement section.

talmi committed:
Version 12

Nits:
- Benoit's comments: https://mailarchive.ietf.org/arch/msg/opsawg/Bb4w80V-uAmor3RFb7sF807lXXs/
- Discussion with Med: https://mailarchive.ietf.org/arch/msg/opsawg/l7q4kaGyvewxM-cKnRFJMSkREV8/

talmi committed:
Updates following WGLC Comments

- Added clarifications about explicit Type I or Type II (Giuseppe).
- Explicit example of in-data-packet combined with active/passive, emphasizing that this is Hybrid Type I (Giuseppe/ Thomas).
- Various nits (Qin).

cpignata commented on draft-ietf-opsawg-oam-characterization.xml:
```diff
@@ -103,16 +103,18 @@
-      <t>This document updates RFC 6291 by adding to the guidelines for the
-      use of the term "OAM". It does not modify any other part of RFC
-      6291.</t>
+      <t>This document updates <xref target="RFC6291"/> by adding to the
```
There's no citations in the Abstract, per RFC Ed guidelines.

cpignata replied:
https://www.rfc-editor.org/rfc/rfc7322.html#section-4.3

cpignata commented on draft-ietf-opsawg-oam-characterization.xml:
```diff
@@ -103,16 +103,18 @@
-      6291.</t>
+      <t>This document updates <xref target="RFC6291"/> by adding to the
+      guidelines for the use of the term "OAM". It does not modify any other
+      part of <xref target="RFC6291"/>.</t>
```
There's no citations in the Abstract, per RFC Ed guidelines.

cpignata replied:
https://www.rfc-editor.org/rfc/rfc7322.html#section-4.3

talmi committed:
Updated In-Data-Packet OAM to Type I

Updated In-Data-Packet OAM to Type I instead of either Type I or Type II.

talmi committed:
Rev 14

Based on Med's review:
https://github.com/boucadair/IETF-Drafts-Reviews/blob/master/2025/draft-ietf-opsawg-oam-characterization-13-rev%20Med.pdf
